### PR TITLE
UCP limitation of UCP disk free monitoring and associated Prom metric

### PR DESCRIPTION
**Pull request by paigehargrave**

### CODE REVIEW

File: datacenter/ucp/2.0/guides/monitor/index.md [hunk 1]
```diff
@@ -65,6 +65,10 @@ If you're accessing this endpoint through a load balancer, you have no way to
 know which UCP manager node is not healthy. So make sure you make a request
 directly to each manager node.
 
+## Monitoring disk usage
+
+Web UI disk usage metrics, including free space, only reflect the Docker managed portion of the filesystem: `/var/lib/docker`.  To monitor the total space available on each filesystem of a UCP worker or manager, you must deploy a monitoring solution, such as  Prometheus Node Exporter.
```
Comment (cameronmartin) [inline datacenter/ucp/2.0/guides/monitor/index.md]:
```suggestion
Web UI disk usage metrics, including free space, only reflect the Docker managed portion of the filesystem: `/var/lib/docker`.  To monitor the total space available on each filesystem of a UCP worker or manager, you must deploy a third party monitoring solution to monitor the operating system.
```

File: datacenter/ucp/2.1/guides/admin/monitor-and-troubleshoot/index.md [hunk 2]
```diff
@@ -59,6 +59,11 @@ way of knowing which UCP manager node is not healthy, since any manager node
 might be serving your request. Make sure you're connecting directly to the
 URL of a manager node, and not a load balancer.
 
+
+## Monitoring disk usage
+
+Web UI disk usage metrics, including free space, only reflect the Docker managed portion of the filesystem: `/var/lib/docker`.  To monitor the total space available on each filesystem of a UCP worker or manager, you must deploy a monitoring solution, such as  Prometheus Node Exporter.
```
Comment (cameronmartin) [inline datacenter/ucp/2.1/guides/admin/monitor-and-troubleshoot/index.md]:
```suggestion
Web UI disk usage metrics, including free space, only reflect the Docker managed portion of the filesystem: `/var/lib/docker`.  To monitor the total space available on each filesystem of a UCP worker or manager, you must deploy a third party monitoring solution to monitor the operating system.
```

File: datacenter/ucp/2.0/guides/monitor/index.md (hunk 1 above)
Comment (cameronmartin) [inline datacenter/ucp/2.0/guides/monitor/index.md]: I'm not sure what the web UI looks like in UCP 2.0 and whether this applies or not.

File: ee/ucp/admin/monitor-and-troubleshoot/index.md [hunk 3]
```diff
@@ -70,6 +70,10 @@ To enable this feature, DTR 2.6 is required and single sign-on with UCP must be
 
 ![example of vulnerability information in UCP](../../images/example-of-vuln-data-in-ucp.png)
 
+## Monitoring disk usage
+
+Web UI disk usage metrics, including free space, only reflect the Docker managed portion of the filesystem: `/var/lib/docker`.  To monitor the total space available on each filesystem of a UCP worker or manager, you must deploy a monitoring solution, such as  Prometheus Node Exporter.
```
Comment (cameronmartin) [inline ee/ucp/admin/monitor-and-troubleshoot/index.md]:
```suggestion
Web UI disk usage metrics, including free space, only reflect the Docker managed portion of the filesystem: `/var/lib/docker`.  To monitor the total space available on each filesystem of a UCP worker or manager, you must deploy a third party monitoring solution to monitor the operating system.
```

File: ee/ucp/admin/configure/collect-cluster-metrics.md [hunk 4]
```diff
@@ -44,6 +44,8 @@ These are metrics about the state of services running on the container platform.
     - Convergence of K8s deployments and Swarm services
     - Cluster load by number of services or containers or pods
 
+**NOTE**: Universal Control Plane web UI disk usage metrics, including free space, only reflect the Docker managed portion of the filesystem: `/var/lib/docker`.  To monitor the total space available on each filesystem of a UCP worker or manager, you must deploy a monitoring solution, such as Prometheus Node Exporter.
```
Comment (cameronmartin) [inline ee/ucp/admin/configure/collect-cluster-metrics.md]:
```suggestion
Web UI disk usage metrics, including free space, only reflect the Docker managed portion of the filesystem: `/var/lib/docker`.  To monitor the total space available on each filesystem of a UCP worker or manager, you must deploy a third party monitoring solution to monitor the operating system.
```

File: datacenter/ucp/3.0/guides/admin/monitor-and-troubleshoot/index.md [hunk 5]
```diff
@@ -64,6 +64,10 @@ URL of a manager node, and not a load balancer. In addition, pinging the
 endpoint with a `HEAD` results in a 404 error code. Use a `GET` request instead.
 
 
+## Monitoring disk usage
+
+Web UI disk usage metrics, including free space, only reflect the Docker managed portion of the filesystem: `/var/lib/docker`.  To monitor the total space available on each filesystem of a UCP worker or manager, you must deploy a monitoring solution, such as  Prometheus Node Exporter.
```
Comment (cameronmartin) [inline datacenter/ucp/3.0/guides/admin/monitor-and-troubleshoot/index.md]:
```suggestion
Web UI disk usage metrics, including free space, only reflect the Docker managed portion of the filesystem: `/var/lib/docker`.  To monitor the total space available on each filesystem of a UCP worker or manager, you must deploy a third party monitoring solution to monitor the operating system.
```

**Comment by paigehargrave** [comment]
@cameronmartin Thank you for the quick responses and help!

**Comment by paigehargrave** [comment]
@david-yu Hi David, can you confirm what UCP versions are affected by this? Thank you in advance!

**Comment by david-yu** [comment]
This affects 3.0.x and 3.1.x not 2.2.x since it does not use Kube. As Cameron suggested I would use the following wording for both versions. 

> Web UI disk usage metrics, including free space, only reflect the Docker managed portion of the filesystem: `/var/lib/docker`.  To monitor the total space available on each filesystem of a UCP worker or manager, you must deploy a third party monitoring solution to monitor the operating system.

**Comment by paigehargrave** [comment]
@david-yu @cameronmartin Thank you!! I've removed the older version changes and incorporated Cameron's wording.

**Pull request merged**